admin/views/access-arrangements/unavailable-access-arrangements.ejs

```diff
@@ -0,0 +1,24 @@
+<%- partial('partials/_gds_breadcrumbs_and_sign_out', { breadcrumbs }) %>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl"><%= title %></h1>
+        <%- partial('partials/_gds_session_expiry') %>
+        <div class="govuk-body govuk-info-message govuk-!-margin-top-9 govuk-!-margin-bottom-9">
+            Currently unavailable as the Multiplication tables check window <% if(availabilityData.checkWindowYear){ %>for <%= availabilityData.checkWindowYear %><% } %> has closed.
```

Comment:
Just checking ... should Multiplication be capitalised?

Reply:
thank you @jon-shipley, good spot.